server/src/main/java/com/linecorp/centraldogma/server/command/StandaloneCommandExecutor.java

```diff
@@ -292,7 +292,8 @@ private CompletableFuture<Void> purgeRepository(PurgeRepositoryCommand c) {
     }
 
     private CompletableFuture<CommitResult> push(RepositoryCommand<?> c, boolean normalizing) {
-        if (c.projectName().equals(INTERNAL_PROJECT_DOGMA) || c.repositoryName().equals(Project.REPO_DOGMA) ||
+        if (c.projectName().equals(INTERNAL_PROJECT_DOGMA) ||
+            Project.internalRepos().contains(c.repositoryName()) ||
```

Comment:
Could you replace all `Project.internalRepos().contains()` with `Project.isInternalRepo()` (a new static method) for brevity? We could introduce some naming rule in the future more easily as well.

Reply:
That's a good point. Addressed. 😉